examples/glfw/Main.hs

```diff
@@ -84,4 +100,41 @@ mainLoop win = do
 
     GLFW.swapBuffers win
 
-    mainLoop win
+    mainLoop win tableRef
+
+mkTable :: IORef [(Integer, String)] -> IO ()
+mkTable tableRef =
+  withTableOpen sortable "MyTable" 3 $ do
+    tableSetupColumn "Hello"
+    tableSetupColumnWith defTableColumnOptions "World"
+
+    withSortableTable \isDirty sortSpecs ->
+      when (isDirty && not (null sortSpecs)) do
+        -- XXX: do your sorting & cache it. Dont sort every frame.
+        putStrLn "So dirty!"
+        print sortSpecs
+        modifyIORef' tableRef . sortBy $
+          foldMap mkCompare sortSpecs
```

Comment:
I think the sort-specs can be in an arbitrary order - because the order-priority is also in the specs given back. Yet i have only seen them in the "correct" order.
Maybe sort in src/DearImGui:1488 with a "assume sorted"-algorithm like bubblesort to make sure?

Reply:
I thought the docs say they have the index matching the entry position:

```c
ImS16 SortOrder; // Index within parent ImGuiTableSortSpecs ...
```

> ... (always stored in order starting from 0, tables sorted on a single criteria will always have a 0 here)